pkg/errors/errors.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 PingCAP, Inc. Licensed under Apache-2.0.
+
+package errors
+
+import "github.com/pingcap/errors"
+
+// BR errors.
+var (
+	ErrUnknown         = errors.Normalize("internal error", errors.RFCCodeText("BR:Common:ErrUnknown"))
+	ErrInvalidArgument = errors.Normalize("invalid argument", errors.RFCCodeText("BR:Common:ErrInvalidArgument"))
+	ErrVersionMismatch = errors.Normalize("version mismatch", errors.RFCCodeText("BR:Common:ErrVersionMismatch"))
+
+	ErrPDUpdateFailed    = errors.Normalize("failed to update PD", errors.RFCCodeText("BR:PD:ErrUpdatePDFailed"))
+	ErrPDLeaderNotFound  = errors.Normalize("PD leader not found", errors.RFCCodeText("BR:PD:ErrUpdatePDFailed"))
+	ErrPDInvalidResponse = errors.Normalize("PD invalid response", errors.RFCCodeText("BR:PD:ErrPDInvalidResponse"))
+
+	ErrBackupUnknown             = errors.Normalize("unknown backup error", errors.RFCCodeText("BR:PD:ErrBackupUnknown"))
```

Comment:
this error is used nowhere, and I don't think it's a good idea to have so many "unknown" errors.